hw/top_earlgrey/top_earlgrey.core

```diff
@@ -29,7 +29,7 @@ filesets:
       - lowrisc:prim:rom_adv
       - lowrisc:ip:rstmgr
       - lowrisc:prim:flash
-      - lowrisc:ip:flash_ctrl:0.1
+      - lowrisc:ip:flash_ctrl
```

Comment:
does verible check core files...? 
LGTM otherwise.

Reply:
no it does not :). let me revert that. I got a weird fusesoc dependency issue which was solved when removing the version suffixes - but it turned out that the version suffixes where not the issue. in the run where this happened, I forgot to purge an old build directory, and that somehow interfered with the build due to the presence of generated core files in there.